aws/tailers/stack_event.go

```diff
@@ -385,3 +367,103 @@ func (s *stackEventTailer) cancelStackUpdate(cfn *awsservices.Cloudformation) er
 	_, err := cfn.CancelUpdateStack(inp)
 	return err
 }
+
+func NewStackEvent(e *cloudformation.StackEvent) stackEvent {
+	return stackEvent{
+		Timestamp:            func() *string { t := e.Timestamp.Format(time.RFC3339); return &t }(),
+		ResourceStatus:       e.ResourceStatus,
+		ResourceType:         e.ResourceType,
+		LogicalResourceId:    e.LogicalResourceId,
+		PhysicalResourceId:   e.PhysicalResourceId,
+		ResourceStatusReason: e.ResourceStatusReason,
+		EventId:              e.EventId,
+	}
+}
+
+// Format reads the struct tag `width:"<width>,<space>"`
+// further marshaling into structured field
+func (s *stackEvent) format(fil filters) []byte {
+	tp := reflect.TypeOf(s).Elem()
```

Comment:
Do you think it would be possible to add table unit tests for that method which seems central? It would help the understanding, document a bit the look of the output, put non regression in place and allow future safe refactoring.

I do not have the answer, I am just wondering.